### ISSUE TITLE
"Preparing spawn area" Stuck

### ISSUE THREAD

**Issue by JustHmm** [ValhelsiaTeam/Valhelsia#91]
Hi, i have issue with my world file. IDK is it related but I wasn't changed the world type when playing in 2.2.2a and played on a "default" type server. Today i switched to 2.2.3. After typing /fml confirm for 3 missing block and couple wall of texts later it stucks at 100% and client always shows "server is still starting" message when trying to connect. 
Server asks "fml confirm" command for missing blocks on every launch. It never able to finish it. 

*I also copied world file from 2.2.2a server to 2.2.3 client and tried to open it as a single player game, after confirming the question for 3 missing item, it stucked again on map generation screen (probably that white square middle of the screen supposed to be generated map area) at 100%

*Changed world type to terraforged and tried again. Its still stucking at "Preparing spawn area: 100%"

Here are the log files of 2.2.3 server: 
[2020-06-20-1.log.gz](https://github.com/ValhelsiaTeam/Valhelsia/files/4807519/2020-06-20-1.log.gz)
(bountiful.log is empty)
[debug.log](https://github.com/ValhelsiaTeam/Valhelsia/files/4807520/debug.log)
[debug-1.log.gz](https://github.com/ValhelsiaTeam/Valhelsia/files/4807527/debug-1.log.gz)
[latest.log](https://github.com/ValhelsiaTeam/Valhelsia/files/4807522/latest.log)

This log in client folder has fatal error message:
[hs_err_pid8608.log](https://github.com/ValhelsiaTeam/Valhelsia/files/4807523/hs_err_pid8608.log)
Log files of the client:
[2020-06-20-1.log.gz](https://github.com/ValhelsiaTeam/Valhelsia/files/4807529/2020-06-20-1.log.gz)
[2020-06-20-2.log.gz](https://github.com/ValhelsiaTeam/Valhelsia/files/4807530/2020-06-20-2.log.gz)
[2020-06-20-3.log.gz](https://github.com/ValhelsiaTeam/Valhelsia/files/4807531/2020-06-20-3.log.gz)
[latest.log](https://github.com/ValhelsiaTeam/Valhelsia/files/4807532/latest.log)

**Comment by JustHmm** [comment]
I think i solved it, here how i solved:
So far i figure it out problem is in world/data folder 
I copied everything except data folder to new server, server started and i did able to connect it. But my game froze and get disconnected.  I saw my everything was there but i coudn't play it.
So i started to transfer data files 
first transferred refine storage and waystone data files:

> minecraft_overworld_refinedstorage_networks.dat
> minecraft_overworld_refinedstorage_nodes.dat
> refinedstorage_disks.dat
> TPDestinations.dat
> waystones.dat
> waystones_NameGenerator.dat

server started. I did able to join and this time i didn't disconnected.
Looks like problem is one of those files: https://prnt.sc/t385l0 and how funny is i just randomly filtered files and solved it. I was thinking "Oh i should transfer those my map files, probably everything important  is in there." no need, lol. Looks like i don't need those 18 dat file. Everything works fine. Probably for a while...
